### DISCUSSION TITLE
Multiple device support?

### DISCUSSION THREAD

**Discussion by ClassickTech** [SideStore/SideStore#608] [General]
Sidestore has been working great on my iPhone 12 for the past few months.

I just got a 13 Pro Max and I keep getting "an internet connection is required to verify trust of the developer" error. Both devices have the same Apple ID. I have no plans on using the 12 anymore so how do I completely switch and get it working?

**Comment by polymo1** [comment]
SideStore doesn't directly have "multiple device support", because it shouldn't need it. However, using multiple devices for SideStore using the same Apple ID has caused issues for multiple users.

Your options are:
- Completely remove SideStore from your iPhone 12, and fully restart the installation process for SideStore on your iPhone 13 Pro Max.
- Use a "burner" Apple ID in SideStore on your iPhone 13 Pro Max. Make sure to use that same burner ID in AltServer.

**Comment by ClassickTech** [comment]
Hello. Thanks for the swift response. As previously mentioned, I no longer have any use for the iphone 12. I did completely remove it from my device, and it still didn't work on my 13 Pro Max. Is there a reset period?

**Comment by polymo1** [comment]
There isn't a set "reset" period, but you could try waiting another week or less and trying again, to wait for the app ids to expire.
